### DISCUSSION TITLE
Your app was rejected

### DISCUSSION THREAD

**Discussion by zeeshanhaider6236dev** [Kyon147/laravel-shopify#45] [General]
**1 App must set security headers to protect against clickjacking.**

There was an error installing your app. The app must be installed to perform the security check. We expected OAuth to be initiated at https://app-security.myshopify.com/admin/oauth/authorize but were redirected to https://app.com/. Your app must request installation immediately after clicking "add app." Apps must request shop access during installation, or reinstallation if the app was previously uninstalled from the shop. [Learn more about authentication in our developer documentation](https://shopify.dev/apps/store/requirements#a-authentication)

**2.App must verify the authenticity of the request from Shopify.**

There was an error installing your app. The app must be installed to perform the security check. We expected OAuth to be initiated at https://app-security.myshopify.com/admin/oauth/authorize but were redirected to https://app.com/. Your app must request installation immediately after clicking "add app." Apps must request shop access during installation, or reinstallation if the app was previously uninstalled from the shop. [Learn more about authentication in our developer documentation](https://shopify.dev/apps/store/requirements#a-authentication)

**Comment by lucastes** [comment]
I think the cause may be incorrect URLs provided in your app's configuration: **App setup -> URLs section**

![image](https://user-images.githubusercontent.com/41701897/213031219-4493a945-f787-4cbc-88fd-03539b0faf1a.png)
